base/replutil.jl

```diff
@@ -150,6 +160,13 @@ function showerror(io::IO, ex::MethodError)
             print(io, "::$typ")
             i == length(arg_types_param) || print(io, ", ")
         end
+        if !isempty(kwargs)
+            print(io, "; ")
+            for (i ,(k, v)) in enumerate(kwargs)
+                print(io, k, "=", v)
```

Comment:
use `show(IOContext(io, :limit => true), v)`, not `print(io, v)` for `v`

(also the `\space,` are in the wrong order on the previous line)